lib/core/catalog/types.ts

```diff
@@ -118,6 +118,16 @@ abstract class BaseCatalogItem implements CatalogItem {
         });
     }
 
+    protected move(original: string, to: string, options?: MoveOptions): Promise<void> {
```

Comment:
It's nice to have this option but I'd prefer these functions to be very specific in their use and have as little options and arguments as possible. So could we perhaps remove the options and just always use `overwrite: true`?

Reply:
Fixed now.